pkgs/top-level/all-packages.nix

```diff
@@ -7924,6 +7924,8 @@ with pkgs;
 
   abi-dumper = callPackage ../development/tools/misc/abi-dumper { };
 
+  abi-compliance-checker = callPackage ../development/tools/misc/abi-compliance-checker { };
```

Comment:
Could you put this before `abi-dumper` to keep them sorted?

Reply:
Ah thanks for the catch; not sure how I missed that (I'm usually very OCD about sorting).